src/pokemon.c

```diff
@@ -6770,6 +6770,131 @@ u16 GetEvolutionTargetSpecies(struct Pokemon *mon, u8 mode, u16 evolutionItem, s
     return targetSpecies;
 }
 
+bool8 CanEvolveByLevelButHasnt(struct Pokemon *mon)
+{
+    int i, j;
+    u16 species = GetMonData(mon, MON_DATA_SPECIES, 0);
+    u32 personality = GetMonData(mon, MON_DATA_PERSONALITY, 0);
+    u8 level = GetMonData(mon, MON_DATA_LEVEL, 0);
+    u16 upperPersonality = personality >> 16;
+    
+    for (i = 0; i < EVOS_PER_MON; i++)
+    {
+        switch (gEvolutionTable[species][i].method)
+        {
+        case EVO_LEVEL_DAY:
+            RtcCalcLocalTime();
```

Comment:
This implies that the multiplier only works for these when the time of day is correct, which Bulbapedia's description doesn't imply.

Reply:
I was under the impression this multiplier is for helping you evolve pokemon that only evolve when leveling up in a specific way, if you missed it before. So it only giving the boost during that specific time makes sense. Not really sure though, so I guess removing the requirement is safest

Reply:
Please confirm how it works before doing so.

Reply:
Confirmed it only checks level and no other condition, apparently.